nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/NuGetUpdater.Core.Test.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>$(CommonTargetFramework)</TargetFramework>
     <OutputType>Exe</OutputType>
     <Nullable>enable</Nullable>
+    <IsTestProject>true</IsTestProject>
```

Comment:
What do we get by adding this?

Reply:
I was throwing stuff at the wall trying to figure out why I couldn't run tests locally. The answer ended up being that I needed to use VS Preview instead of  2022.